packages/odf/components/kms-config/name-address-port.tsx

```diff
@@ -112,20 +135,28 @@ export const NameAddrPort: React.FC<NameAddrPortProps> = ({
             'ocs-install-kms__form-port',
             `${className}__form-body--small-padding`
           )}
-          helperTextInvalid={validatePortMessage()}
-          validated={isValid(kmsState.port.valid)}
           isRequired
         >
           <TextInput
             value={kmsState.port.value}
-            onChange={setAddressPort}
+            onChange={(_event, port: string) => setAddressPort(port)}
             type="text"
             id="kms-address-port"
             name="kms-address-port"
             isRequired
             validated={isValid(kmsState.port.valid)}
             data-test="kms-address-port-text"
           />
+          <FormHelperText>
+            <HelperText>
+              <HelperTextItem
+                variant={isValid(kmsState.port.valid) ? 'default' : 'error'}
```

Comment:
```suggestion
                variant={getValidatedProp(!kmsState.port.valid)}
```
or just isValid(kmsState.port.valid)  is enough, isValid already returns  'default' : 'error'